src/modules/Home/components/Content/index.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react';
+import {
+  Container,
+  Grid,
+  Typography,
+  Paper,
+  Divider,
+  TextField,
+  IconButton,
+} from '@material-ui/core/';
+import { SearchOutlined } from '@material-ui/icons';
+import Article from '../Articles';
+import articles from '../../constants';
+import Chips from '../../../shared/Chips';
+import classes from '../../../shared/Chips/Chips.module.css';
+
+import './Content.css';
+
+function Content() {
+  const [chipsArr, setChipsArr] = useState({ search: '', filter: [] });
```

Comment:
const [appliedFilter, setAppliedFilter] = useState({ search: '', categories: [] });

Reply:
renamed vars for better readability.